app/helper.js

```diff
@@ -505,6 +509,91 @@ class YoutubeGrabberHelper {
     }
   }
 
+  parseHomeItem(item, channelInfo) {
+    let shelfName = ''
+    let shelfUrl = null
+    let items = []
+    let type = 'video'
+    if ('richSectionRenderer' in item) {
+      const shelf = item.richSectionRenderer.content.richShelfRenderer
+      shelfName = shelf.title.simpleText
+      type = 'verticalVideoList'
+      items = shelf.contents.map(shelfContent => {
+        return this.parseVideo(shelfContent.richItemRenderer.content, channelInfo)
+      })
+    } else if ('richItemRenderer' in item) {
+      items = [this.parseVideo(item.richItemRenderer.content, channelInfo)]
+    } else if ('itemSectionRenderer' in item) {
+      const shelf = item.itemSectionRenderer.contents[0].shelfRenderer
+
+      if ('runs' in shelf.title) {
+        const title = shelf.title.runs[0]
+        if ('navigationEndpoint' in title) {
+          shelfUrl = title.navigationEndpoint.commandMetadata.webCommandMetadata.url
+        }
+        shelfName = title.text
+      } else {
+        shelfName = shelf.title.simpleText
+        shelfUrl = shelf.endpoint.commandMetadata.webCommandMetadata.url
+      }
+      if (shelfUrl === null) {
+        let shelfRenderer
+        if ('expandedShelfContentsRenderer' in shelf.content) {
+          type = 'verticalVideoList'
+          shelfRenderer = shelf.content.expandedShelfContentsRenderer
+          items = shelfRenderer.items.map(video => {
+            return this.parseVideo(video, channelInfo)
+          })
+        } else {
+          type = 'playlist'
+          shelfRenderer = shelf.content.horizontalListRenderer
+          items = shelfRenderer.items.map(pl => {
+            return this.parsePlaylist(pl, channelInfo)
+          })
+        }
+      } else if (new RegExp(/\?list=/).test(shelfUrl)) {
```

Comment:
you don't need to wrap `/regex/` in `new RegExp()`, as `/expression/` is the literal notation to create a `RegExp` object. It's like doing `new String('the string')`.